file-formats/enhs/type/zif_seef_aff_enhs_v1.intf.abap

```diff
@@ -0,0 +1,223 @@
+    "! Information about this BAdI definition
+    BEGIN OF ty_badi_def,
+      "! <p class="shorttext">Name of the BAdI Definition</p>
+      "! Name of the BAdI definition
+      "! $required
+      name               TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Description of the BAdI Definition</p>
+      "! Description of the BAdI definition
+      "! $required
+      description        TYPE c LENGTH 255,
+      "! <p class="shorttext">Interface of the BAdI Definition</p>
+      "! Interface of the BAdI definition
+      "! $required
+      interface          TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Instantiation of the BAdI Definition</p>
+      "! Instantiation of the BAdI definition
+      instantiation      TYPE ty_instantiation,
+      "! <p class="shorttext">Single- or Multiple Use BAdI</p>
+      "! Indicator whether the BAdI definition is single- or multiple usable
+      multiple_use       TYPE abap_bool,
+      "! <p class="shorttext">SAP Internal</p>
+      "! SAP internal
+      sap_internal       TYPE abap_bool,
+      "! <p class="shorttext">Example Classes of the BAdI Definition</p>
+      "! Example classes of the BAdI definition
+      example_classes    TYPE ty_example_classes,
+      "! <p class="shorttext">Use Fallback Class</p>
+      "! Use fallback class in case no implementation is executed
+      use_fallback_class TYPE abap_bool,
+      "! <p class="shorttext">Name of the Default/Fallback Class</p>
+      "! Name of the default/fallback class
+      fallback_class     TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Limited Filter Use</p>
+      "! Limited filter use
+      filter_limitation  TYPE abap_bool,
+      "! <p class="shorttext">Documentation Id</p>
+      "! Documentation Id
+      documentation_id   TYPE c LENGTH 60,
+      "! <p class="shorttext">Amdp BAdI</p>
+      "! BADI is an AMDP BAdI
+      amdp               TYPE abap_bool,
+      "! <p class="shorttext">Filters of the BAdI Definition</p>
+      "! Filters of the BAdI definition
+      filters            TYPE ty_badi_def_filters,
+    END OF ty_badi_def.
+
+  TYPES ty_badi_defs TYPE STANDARD TABLE OF ty_badi_def WITH DEFAULT KEY.
+
+  TYPES:
+    BEGIN OF ty_configuration,
+      "! <p class="shorttext">SAP Internal</p>
+      "! SAP internal
+      sap_internal TYPE abap_bool,
+      "! <p class="shorttext">BAdI Enhancement Spot Tool Type</p>
+      "! Tool type of the BAdI Enhancement Spot
+      "! $required
+      tool         TYPE ty_tool,
+    END OF ty_configuration,
+
+    "! <p class="shorttext">ENHS Object Type</p>
+    "! Object type ENHS
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Schema</p>
+      "! Format version
+      "! $required
+      schema           TYPE string,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header           TYPE zif_aff_types_v1=>ty_header_60,
```

Comment:
Enhancement Editor allows to enter 100 characters as description. I guess we have to introduce a new header :(

Reply:
I added a new header zif_aff_types_v1=>ty_header_100